src/nlr/DeepPolyLeakyReLUElement.cpp

```diff
@@ -0,0 +1,273 @@
+ /*********************                                                        */
+/*! \file DeepPolyLeakyReLUElement.cpp
+ ** \verbatim
+ ** Top contributors (to current version):
+ **   Haoze Andrew Wu
+ ** This file is part of the Marabou project.
+ ** Copyright (c) 2017-2019 by the authors listed in the file AUTHORS
+ ** in the top-level source directory) and their institutional affiliations.
+ ** All rights reserved. See the file COPYING in the top-level source
+ ** directory for licensing information.\endverbatim
+ **
+ ** [[ Add lengthier description here ]]
+
+**/
+
+#include "DeepPolyLeakyReLUElement.h"
+#include "FloatUtils.h"
+
+namespace NLR {
+
+DeepPolyLeakyReLUElement::DeepPolyLeakyReLUElement( Layer *layer )
+{
+    _layer = layer;
+    _size = layer->getSize();
+    _layerIndex = layer->getLayerIndex();
+    _alpha = layer->getAlpha();
```

Comment:
again, I think slope is better than alpha.
Are we sure all leakyRelus in the layer have the same slope?

Reply:
rename slope to _alpha in the DeepPoly LeakyReLUElement.